### ISSUE TITLE
Copy 3d selector gets too much

### ISSUE THREAD

**Issue by BlappoSpawn** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#212]
**Describe the bug**
i was selecting a house but got alot more.

**To Reproduce**
1. im not sure yet.......

**Screenshots/Videos/Twitch Clips**
https://youtu.be/EayPrD28CZg

**Additional context**
hmm

**Comment by BlappoSpawn** [comment]
https://youtu.be/9GQIhJLB1uU

ummm still do not know what to do to repro... but i keep accidentally doing it.

**Comment by BlappoSpawn** [comment]
i have thoughts... mostly guessing... but do lush grass tiles clip... dragging them out against eachother i think i had placement issues... clipping would make it select more... idk.... maybe not

**Comment by BlappoSpawn** [comment]
ok here is me selecting the same piece a few times.   The only time it grabs too much is when i drop the selector tool below the lowest BP 
https://youtu.be/xx1744fZpaA

This is odd, because i got in the habit of doing this because it often stops the copy paste bug where some tiles are missing from a slab.  

perhaps specific tiles obj have an unseen edge extruded out pixel thin? idk something is happening under the board or perhaps the tool doesnt like being below the board..

**Comment by LuPro** [comment]
Might be related to #51, I've seen one or two other issues kind of related to that, but can't quite remember (and this one is the closest to it)

**Comment by BlappoSpawn** [comment]
non-update.  still cant reproduce consistently. i spent some time trying but its not piece type or even selection size...   my specific set of tiles on the map i can pick out ways to trigger and not the bug... but still nothing is consistent between the tiles or selection size or anything else i notice other than tiles are on base BP
